.github/workflows/esti.yaml

```diff
@@ -653,6 +654,64 @@ jobs:
         working-directory: test/spark
         run: docker-compose logs --tail=1000 lakefs
 
+  metadata-client-gc-spark3-on-azure:
+    name: Test lakeFS metadata client GC with Spark 3.x on Azure
+    needs: [deploy-image, build-spark3-metadata-client]
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        spark:
```

Comment:
Do we want to add other versions of spark (3.1.2, 3.0.2)?

Reply:
I removed it intentionally, since we support for now only in hadoop3, and this is the relevant version for it. I'll open a new issue for testing also other versions.